app/views/layouts/application.html.slim

```diff
@@ -6,7 +6,7 @@ html.govuk-template lang='en'
     = render 'layouts/analytics_header'
     = render 'layouts/hotjar_tracking'
     = tag.meta content: 'width=device-width,initial-scale=1', name: 'viewport'
-    = tag.meta name: 'robots', content: 'noindex,nofollow'
+    = tag.meta name: 'robots', content: 'noindex,nofollow' unless Rails.env.production?
```

Comment:
All our deployments are production; so, without creating another Rails env, is it worth leaving the tag in for the other workspaces? @ebrett